rust/main/hyperlane-base/src/settings/signers.rs

```diff
@@ -5,15 +5,18 @@ use ethers::utils::hex::ToHex;
 use eyre::{bail, Context, Report};
 use hyperlane_core::{AccountAddressType, H256};
 use hyperlane_sealevel::Keypair;
+use hyperlane_ton::TonSigner;
 use rusoto_core::Region;
 use rusoto_kms::KmsClient;
+use tonlib_core::wallet::WalletVersion;
+
 use tracing::instrument;
 
 use super::aws_credentials::AwsChainCredentialsProvider;
 use crate::types::utils;
 
 /// Signer types
-#[derive(Default, Debug, Clone)]
+#[derive(Default, Debug, Clone)] /////
```

Comment:
I wonder why we need the comment in the end of the line?

Reply:
Fixed